doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -120,6 +120,7 @@ Added:
 Updated:
 
 * The power management support in modules is now enabled by default when the :kconfig:`CONFIG_CAF_PM_EVENTS` Kconfig option is enabled.
+* The :ref:`caf_power_manager` now has a dependency on :kconfig:`CONFIG_PM_POLICY_APP`, which is required by the application that is using the :ref:`caf_power_manager` to link.
```

Comment:
Typo in commit message:
configiration -> configuration

Reply:
ok